build-config/checkstyle/build/suppressions.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No need to commit this file. It will get automatically downloaded from https://github.com/ballerina-platform/module-ballerinax-azure.functions/pull/1/files#diff-446377c3ecf1a1a206f1a4a3f5716024R22-R30

Reply:
Noted and removed.